### ISSUE TITLE
Print in different Line in Zeppelin

### ISSUE THREAD

**Issue by dimoibiehg** [tqdm/tqdm#632]
- [x] I have visited the [source website], and in particular
  read the [known issues]
- [x] I have searched through the [issue tracker] for duplicates
- [x] I have mentioned version numbers, operating system and
  environment, where applicable:
  ```python
  import tqdm, sys
  print(tqdm.__version__, sys.version, sys.platform)
  >>> 4.24.0 3.5.2 (default, Nov 23 2017, 16:37:01) 
  >>> [GCC 5.4.0 20160609] linux
  ```

The issue is the library does not work in zeppelin (>= 0.7). Indeed, it just write each progress bar in a differerent line.

  [source website]: https://github.com/tqdm/tqdm/
  [known issues]: https://github.com/tqdm/tqdm/#faq-and-known-issues
  [issue tracker]: https://github.com/tqdm/tqdm/issues?q=

**Comment by vaibhawc** [comment]
The same thing happens with Atom Integrated with Hydorgen.

**Comment by chengs** [comment]
@dimoibiehg recently I don't have too much time work on this issue. Does Zeppelin work like Jupyter? PR or additional information regarding this issue is highly welcomed.

**Comment by dimoibiehg** [comment]
@chengs  No. Zeppelin is different here from Jupyter. As I've mentioned, the progress bar in Zeppelin can be modified through a context. You can find more from [here](https://github.com/apache/zeppelin/pull/2454). I've not more info about this.
